webapp/package.json

```diff
@@ -24,7 +24,7 @@
     "@angular/platform-browser": "^7.2.15",
     "@angular/platform-browser-dynamic": "^7.2.15",
     "@angular/router": "^7.2.15",
-    "@c4dt/cothority": "^3.1.15",
+    "@dedis/cothority": "^3.1.3",
```

Comment:
Wanted? Was introduced by #57 and `@dedis/cothority` is not NPM updated yet

Reply:
Oups - I thought that was a leftover from me...